From bf6449901bd0745d99ff387cb1a72c930e6c6f67 Mon Sep 17 00:00:00 2001
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Wed, 8 Sep 2021 10:47:23 +0200
Subject: [PATCH 5/8] arm: dts: imx6ull-colibri-emmc: Enable Display interfaces

Toradex Easy Installer does not support dedicated colibri-imx6ull-emmc
target. Overlays can't be clearly applied to NAND device tree.

Enable lcdif, backlight, panel-dpi and pwm interfaces by default.

Related-to: TEI-921
Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
---
 .../boot/dts/imx6ull-colibri-emmc-nonwifi.dtsi   | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-colibri-emmc-nonwifi.dtsi b/arch/arm/boot/dts/imx6ull-colibri-emmc-nonwifi.dtsi
index ea238525d5c0..147f1f8923e0 100644
--- a/arch/arm/boot/dts/imx6ull-colibri-emmc-nonwifi.dtsi
+++ b/arch/arm/boot/dts/imx6ull-colibri-emmc-nonwifi.dtsi
@@ -185,3 +185,19 @@ &usdhc2 {
 	vmmc-supply = <&reg_module_3v3>;
 	status = "okay";
 };
+
+&backlight {
+	status = "okay";
+};
+
+&lcdif {
+	status = "okay";
+};
+
+&panel_dpi {
+	status = "okay";
+};
+
+&pwm4 {
+       status = "okay";
+};
-- 
2.35.3

